### PR TITLE
Beta tool install interface improvements

### PR DESCRIPTION
**Pull request by martenson**

### CODE REVIEW

File: client/galaxy/scripts/mvc/toolshed/categories-view.js [hunk 1]
```diff
@@ -60,44 +55,44 @@ var ToolShedCategories = Backbone.View.extend({
...
-            '<td><%= category.get("description") %></td>',
-            '<td><%= category.get("repositories") %></td>',
-            "</tr>",
-            "<% }); %>",
-            "</table>",
-            "</div>",
-            "</div>"
-        ].join("")
-    )
+    templateCategoryList: function() {
```
Comment (guerler) [inline client/galaxy/scripts/mvc/toolshed/categories-view.js:58]: Can we use multi-line strings here?

File: lib/galaxy/webapps/galaxy/api/toolshed.py [hunk 2]
```diff
@@ -167,14 +168,17 @@ def show(self, trans, **kwd):
...
+            for category in json.loads(util.url_get(url)):
+                api_url = web.url_for(controller='api/tool_shed',
+                                      action='category',
+                                      tool_shed_url=urlquote(tool_shed_url),
+                                      category_id=category['id'],
+                                      qualified=True)
+                category['url'] = api_url
+                categories.append(category)
+        except Exception:
+            raise exceptions.ObjectNotFound('The specified Toolshed is not responding.')
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/toolshed.py]:
```suggestion
            raise exceptions.ObjectNotFound("Tool Shed %s is not responding." % tool_shed_url)
```

File: client/galaxy/scripts/mvc/toolshed/categories-view.js (hunk 1 above)
Comment (martenson) [reply client/galaxy/scripts/mvc/toolshed/categories-view.js:58]: converted all the templates to multiline, thanks!

**Comment by martenson** [comment]
I believe the failing test is unrelated. Ping @guerler

**Comment by martenson** [comment]
Thanks for the merge!

**Pull request merged**